day_7/ceasar_cipher.rb

```diff
@@ -0,0 +1,36 @@
+class CeasarCipher
+  attr_accessor :phrase, :shift_value
+  attr_reader :new_cipher
+
+  def initialize()
+    @plain_alphabet = ("A".."Z").to_a
+    @cipher_alphabet = Array.new
+    @cipher_index = Array.new
+    @new_cipher = Array.new
```

Comment:
It is fine to set an attribute to be an empty array, and in fact, that may be the more common pattern. `@new_cipher = []` is just fine.